src/server.c

```diff
@@ -150,62 +116,16 @@ void
 sig_handler(int sig) {
     if (force_exit)
         exit(EXIT_FAILURE);
-
-    char sig_name[20];
-    get_sig_name(sig, sig_name);
-    lwsl_notice("received signal: %s (%d), exiting...\n", sig_name, sig);
+    lwsl_notice("received signal: %s (%d), exiting...\n", strsignal(sig), sig);
```

Comment:
Yes, it makes sense, please revert 😄 .

Some process may not exit immediately after received the stop signal, this message is useful to notice user ttyd has received stop request.

```
[2017/08/05 14:12:37:4463] NOTICE: received signal: SIGINT (2), exiting...
[2017/08/05 14:12:37:4464] NOTICE: send ^C to force exit.
```